pom.xml

```diff
@@ -405,6 +419,15 @@
                 </plugins>
             </reporting>
         </profile>
+        <profile>
```

Comment:
So I've isolated the reason our unit tests are breaking in J11 down to the injected tests. I haven't gone over them thoroughly, but they appear mostly to be for things like Jelly formatting https://groups.google.com/g/jenkinsci-dev/c/POdTEQKq89s. They're a bit slower to run, and so I'm now imagining them running in a separate profile- which means for full tests coverage you have to maven twice. Not ideal.
Very open to feedback on this very very hacky solution.

Reply:
I think to find a true fix for this we'll need to reach out to the Jenkins community, so this is a stop-gap to unblock us from releasing.